test/functional/Valhalla/src/org/openj9/test/lworld/ValhallaUtils.java

```diff
@@ -25,12 +25,12 @@
 
 public class ValhallaUtils {
 	/**
-	 * Currently value type is built on JDK24, so use java file major version 68 for now.
+	 * Currently value type is built on JDK25, so use java file major version 69 for now.
 	 * If moved this needs to be incremented to the next class file version. The check in j9bcutil_readClassFileBytes()
```

Comment:
nit: Since you're here this would be more helpful as a reminder to update VALUE_TYPES_MAJOR_VERSION. That code has been refactored since this was written.

Reply:
Updated the comments.